pom.xml

```diff
@@ -182,7 +182,7 @@ limitations under the License.
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
-                    <version>3.0.1</version>
+                    <version>3.3.1</version>
```

Comment:
@dermot-hardy Apologies, send the last PR a bit too early, didn't need the workaround after updating to the latest version of the plugin.